movePlayerScript.js

```diff
@@ -0,0 +1,40 @@
+var storedKeyPress = 65;
+
```

Comment:
comment needed

Reply:
its an event listener, i don't think it needs it